examples/3d/tonemapping.rs

```diff
@@ -224,7 +224,7 @@ fn setup_color_gradient_scene(
     camera_transform: Res<CameraTransform>,
 ) {
     let mut transform = camera_transform.0;
-    transform.translation += transform.forward();
+    transform.translation += *transform.forward();
```

Comment:
One way to avoid the deref here would be to simply implement `Add<Direction3d>` for `Vec3`. That can be done in a follow-up though

Reply:
I would find this a little odd, from an API perspective. If we include direction in the vector algebra we might as well just call it `UnitVector`.